main.css

```diff
@@ -0,0 +1,40 @@
+.main-nav {
+    background-color: #3590F3;
+    color: #F5F5F5;
+    
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    padding-left: 10px;
+}
+
+.nav-links {
+    list-style: none;
+    font-family: "proxima-nova","Helvetica Neue",Helvetica,Arial,sans-serif; /*Figure out a new font*/
+
```

Comment:
Very small thing but there is an unnecessary space here between `font-family` and `display` (unless that was intentional).